src/test/java/subway/StationApiRequester.java

```diff
@@ -0,0 +1,72 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import subway.station.StationResponse;
+
+public final class StationApiRequester {
+
+    private StationApiRequester() {
+    }
+
+    public static void updateStation(Long id, String name) {
+        final Map<String, String> params = new HashMap<>();
+        params.put("name", name);
+
+        RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().put("/stations/" + id)
```

Comment:
```suggestion
            .when().put("/stations/{stationId}", id)
```

위와 같은 형태로 하면 가독성이 더 좋은 것 같아요 😄